src/test/java/bio/terra/service/dataset/flight/inheritsteward/InheritStewardSetFlagStepTest.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.service.dataset.flight.inheritsteward;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.dataset.DatasetDao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class InheritStewardSetFlagStepTest {
+
+  @Mock private DatasetDao datasetDao;
+  private UUID datasetId;
+  private boolean enableInheritSteward;
+  private InheritStewardSetFlagStep step;
+  private FlightContext context;
+
+  @BeforeEach
+  void setUp() {
+    datasetId = UUID.randomUUID();
+    enableInheritSteward = true;
+    step = new InheritStewardSetFlagStep(datasetDao, datasetId, enableInheritSteward);
+    context = mock(FlightContext.class);
+  }
+
+  @Test
+  void testDoStepSuccess() {
+    when(datasetDao.setInheritSteward(datasetId, enableInheritSteward)).thenReturn(true);
```

Comment:
What is the purpose of using `enableInheritSteward` in these test methods? Its value is always `true`.
```suggestion
    when(datasetDao.setInheritSteward(datasetId, true)).thenReturn(true);
```